src/hooks/requests/useProjects.ts

```diff
@@ -131,6 +131,18 @@ export const useUpdateProject = ({ id }: { id?: string }) => {
           return { data: newData }
         }
       )
+      map(data.sub_projects, (subProject) =>
+        queryClient.setQueryData(
+          ['subprojects', subProject.id],
+          (oldData?: ProjectsResponse) => {
```

Comment:
**[blocker]**

Not sure about the typing here. oldData type should be SubProjectResponse